src/main/java/org/datadog/jenkins/plugins/datadog/steps/TestVisibility.java

```diff
@@ -18,6 +18,13 @@ public class TestVisibility implements Serializable {
     public TestVisibility() {
     }
 
+    public TestVisibility(boolean enabled, String serviceName, Collection<TracerLanguage> languages, Map<String, String> additionalVariables) {
```

Comment:
Should we name this `TestOptimization` or `TestOptimizationConfig` class? (Given the fact that's just a model class with getters/setters that keeps the config of Test Optimization)